.github/workflows/pull_request.yml

```diff
@@ -6,15 +6,13 @@ jobs:
   go-checks:
     runs-on: ubuntu-latest
     steps:
-    - name: Install Go
-      uses: actions/setup-go@v2
-      with:
-        go-version: '1.21'
-    - name: Checkout code
-      uses: actions/checkout@v2
-    - name: gofmt
-      uses: gametimesf/github-actions/go/fmt@v0.8.0
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - name: Test
-      run: go test -cover -race ./...
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.21"
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: gofmt
+        run: if [ "$(gofmt -l -d -e . | wc -l)" -gt 0 ]; then exit 1; fi
```

Comment:
oh sorry, does this log the actual files that needed formatting? why not just use what the previous action did? 

https://github.com/gametimesf/github-actions/blob/master/go/fmt/entrypoint.sh#L8

Reply:
It doesn't. Sounds like there was some miscommunication when I asked you if you wanted what the previous action did still or just to error. You noted you don't need it to create a comment. To clarify, you still want it to show the files that are in question? Do you want it to show the places still where the errors are as well, or does just the file names suffice?

Reply:
ah, sorry about that. i think file names suffice 👍 . but i think i miscommunicated that no comment on the PR itself is necessary.

Reply:
we have several checks elsewhere that don't actually comment on the PR and you have to go look at the action log to figure out what, exactly, failed. in this particular case, though, it simply saying "hey the go fmt action failed" it should be pretty obvious that the fix is to go run `go fmt`.

Reply:
that's fine. i didn't think it would be a big ask to get the files outputted. the last action commented on the PR so i figured a good compromise would be to output in the action logs.

Reply:
you have to make a curl request to post a comment and that just feels like too much for an inline action in one repo that gets very little traction in the first place

Reply:
How does this look @jeffatgametime ? 
![Screenshot 2024-03-08 at 5 20 17 PM](https://github.com/gametimesf/testy/assets/159076325/ccd4fa53-51ad-4827-87c1-23ff7419dc94)

Reply:
It's printed in the action still of what the error(s) are if there are any

Reply:
yupp perfecto. @ajanata i was just suggesting the above, not commented on the PR (agree that's a PITA).

Reply:
sweet! I'll put this in then